pkg/catalogv2/helm/release.go

```diff
@@ -19,12 +19,14 @@ import (
 )
 
 var (
-	ErrNotHelmRelease = errors.New("not helm release")
-	magicGzip         = []byte{0x1f, 0x8b, 0x08}
+	ErrNotHelmRelease = errors.New("not helm release") // error for when it's not a helm release
+	magicGzip         = []byte{0x1f, 0x8b, 0x08}       // gzip magic header
 )
 
+// IsNamespaced Alias for func(gvk schema.GroupVersionKind) bool
 type IsNamespaced func(gvk schema.GroupVersionKind) bool
 
+// IsLatest checked if the received release is the latest one
```

Comment:
// IsLatest checks if the received releaseSpec is the latest version of the
// chart installed by comparing the release.Version to all the release helm objects.